### ISSUE TITLE
rxjs-no-exposed-subjects could allow using an EventEmitter

### ISSUE THREAD

**Issue by chrisguttandin** [cartant/rxjs-tslint-rules#76]
Hi, I would like to propose to make the `rxjs-no-exposed-subjects` rule configurable. I generally agree that a Subject should not be exposed but with one notable exception.

In Angular it is very common to have code like this.

```typescript
import { Component, EventEmitter, Output } from '@angular/core';

@Component({ /* ... */ })
export class ComponentWithAnOutput {

    @Output() public clicked = new EventEmitter<void>();

    public emitClickedEvent (): void {
        this.clicked.emit();
    }

}
```

The problem is that `EventEmitter` extends `Subject`. The `emit()` function is basically the same as the `next()` function. https://angular.io/api/core/EventEmitter

Would it be an option to allow an `EventEmitter` to be exposed as a public property?

**Comment by cartant** [comment]
Yeah. Begrudgingly, it should be an exception. IMO, `EventEmitter` should not have extended `Subject`.

**Comment by chrisguttandin** [comment]
Many thanks for the quick fix.
